devtools/conda-envs/test_env.yaml

```diff
@@ -24,6 +24,7 @@ dependencies:
   - mmligner
   # theseus
   - muscle
+  - clustalo
   - theseus
```

Comment:
You are installing these three via `bioconda` correct? Since we currently have those not included in our `opencadd` conda package, could you please open a PR describing the installation of those? 

In the future, we will need to include instructions in our documentation.

Reply:
Theseus is in conda-forge. The other two are in bioconda.